### PR TITLE
Use insta for the integration tests

### PR DESCRIPTION
**Pull request by lu-zero**
The rust repo is removed since it will be replaced with a stable codebase for the rust case.

### CODE REVIEW

**Comment by lu-zero** [comment]
The windows tests are still problematic but would be good to investigate them at a later time.

File: src/spaces.rs [hunk 1]
```diff
@@ -164,8 +164,10 @@ impl FuncSpace {
                 node.object().end_position().row + 1,
             ),
         };
+
         Self {
-            name: T::get_func_space_name(node, code).map(|name| name.to_string()),
+            name: T::get_func_space_name(node, code)
+                .map(|name| name.split_whitespace().collect::<Vec<_>>().join(" ")),
```
Comment (marco-c) [inline src/spaces.rs:170]: Why was this necessary?

**Review by marco-c — APPROVED** [review]
